apps/main/lib/main/entities/article.rb

```diff
@@ -0,0 +1,20 @@
+require "redcarpet"
+require "rom/struct"
+
+module Main
+  module Entities
+    class Article < ROM::Struct
+      def body_html
```

Comment:
I wonder if this could be better served by some sort of view object. Having an Entity be aware of HTML seems to be a cross cutting concern.